README.md

```diff
@@ -149,6 +149,49 @@ juju run postgresql-k8s/leader restore backup-id=YYYY-MM-DDTHH:MM:SSZ --wait 5m
 ```
 More details found [here](https://charmhub.io/postgresql-k8s/docs/h-restore-backup).
 
+## Scaling
+The Superset charm consists of 3 services (server, worker and beat), the server and worker services are independently scalable.
```

Comment:
maybe you could say here that: these services are deployed as separate applications, which scale independently

... i understood you were talking about applications only when you gave the example in line 159

Reply:
Updated to make it more clear :)